portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Conversation;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comments")
+public class CommentServlet extends HttpServlet {
+  private Conversation conversation = new Conversation();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+    String json = new Gson().toJson(conversation);
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String username = getUsername(request);
+    String commentText = getComment(request);
+    Comment comment = new Comment(username, commentText);
+
+    conversation.addComment(comment);
+    response.sendRedirect("/community.html");
+  }
+
+  private String getUsername(HttpServletRequest request) {
+    String username = request.getParameter("username");
+
+    // Sanatize input here
```

Comment:
If this is a todo to add sanitization logic, start the comment with // TODO:

Reply:
Added todo comment